.github/ISSUE_TEMPLATE/doc_issue.yml

```diff
@@ -34,6 +34,4 @@ body:
     label: Would you want to open a pull request to fix it?
     options:
       - 'yes'
-      - 'no'
-      
-           
+      - 'no'
```

Comment:
just a thought on this one - do you think we risk that people think they might look bad if they answer "no"?

maybe we could rephrase, so as to not require an answer, but instead just write "if you'd be interested in opening a pull request to fix this, please let us know!"

Reply:
I changed it. We have the same question on  the feature request template. I was going to change that too, but I'm not sure. Can you confirm please? If yes, on the same PR?

Reply:
Yeah, sure

Reply:
pronto